packages/compass-aggregations/src/modules/pipeline-builder/pipeline-parser/utils.ts

```diff
@@ -1,8 +1,8 @@
 import babelGenerate from '@babel/generator';
 import type { Node } from '@babel/types';
-import prettier from 'prettier';
 import _parseEJSON, { ParseMode } from 'ejson-shell-parser';
 import type { Document } from 'mongodb';
+import { prettify } from '@mongodb-js/compass-editor';
```

Comment:
Eventually we will probably move the low-level parsing method that uses ejson-shell-parser from this file to the editor package too, we currently have a bunch of places where this stuff is imported and used directly, editor seems like a good place for me to put all of them (you'd usually have a dependency on the editor anyway when you need to do parsing)